### ISSUE TITLE
🐛 (vscode-extension) Cannot add suppression comment in JSX

### ISSUE THREAD

**Issue by JoostKersjes** [rome/tools#1483]
## Environment

```
$ yarn rome rage --summary
yarn run v1.22.10
$ /home/joost/projects/personal/kersjes.dev/node_modules/.bin/rome rage --summary
Environment Variables
  Object {
    COLORTERM: "truecolor"
    LANG: "C.UTF-8"
    TERM: "xterm-256color"
    TERM_PROGRAM: "vscode"
  }

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  16.1.0

Platform
  linux x64 4.19.104-microsoft-standard

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 24
    columns: 319
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/home/joost/projects/personal/kersjes.dev"
    realCwd: "/home/joost/projects/personal/kersjes.dev"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 60_071_936
      pid: 20_905
      uptime: 0.8430033
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 60_071_936
        ownedBytes: 0
        ownedFileCount: 0
        pid: 20_905
        uptime: 0.8429048
      }
    ]
  }


Done in 0.96s.
```

## Steps To Reproduce

I'm running VSCode on Windows 10 as a remote window to WSL2 Ubuntu-18.04.
This is in a Next.js project.

1. Create any JSX linting error. For example `<Component {...pageProps} />`.
2. Open the "Quick Fix" menu (default shortcut is `Ctrl` + `.`) when the cursor is at the linting error.
3. Select the "Add suppression comment" option.

For me, this automatically opened the VSCode Rome Output. This was the message I got:

```
[Error - 10:21:02 PM] Request workspace/executeCommand failed.
  Message: Connection died

 project-rome/@internal/events/Bridge.ts:351:20 internalError/request ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Connection died

  1. ___R$project$rome$$internal$core$common$bridges$WorkerBridge_ts$default.end (project-rome/@internal/events/Bridge.ts:351:20)
  2. ___R$project$rome$$internal$core$server$WorkerManager_ts$default.end (project-rome/@internal/core/server/WorkerManager.ts:106:11)
  3. ___R$project$rome$$internal$core$server$Server_ts$default.onFatalError (project-rome/@internal/core/server/Server.ts:335:21)
  4. onFatalError (project-rome/@internal/core/server/WorkerManager.ts:287:17)
  5. Timeout.<anonymous> (project-rome/@internal/events/Bridge.ts:178:7)

  ℹ Error occurred while requesting getBuffer for kersjes.dev/src/pages/_app.tsx


  Code: -32603
```

## Expected Results

The suppression comment should have been added. Expected code for this case would be:
```js
{/* rome-ignore lint/jsx/noPropSpreading: nextjs _app.tsx convention */}
```

**Comment by ematipico** [comment]
Bug confirmed. At the moment the [visitor that we create to generate the suppression](https://github.com/rome/tools/blob/984c93cce0684f712f5adb14e91e85e382ba1add/internal/compiler/suppressions.ts#L66-L83), is not able to pick the correct comment inside a JSX file.

Somehow, we have to instruct this visitor to handle also cases when parsing JSX files. This would require navigating a bit the AST and generate the suppression for the correct node.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
